src/options.rs

```diff
@@ -57,6 +57,9 @@ pub struct ColumnOptions {
 	/// Allows for skipping additional key hashing.
 	pub uniform: bool,
 	/// Use reference counting for values.
+	///
+	/// Reference counting do not enforce immediate removal
+	/// and user should not check for missing value.
```

Comment:
What do you mean here? As soon as reference count reaches 0 the value is removed, isn't it?

Reply:
Not in the commit overlay, there we simply skip all removal.

Reply:
(I think our rc usecase is related to rc + uniform, which make sense for trie nodes).
Thinking about it I almost thought of forbidding rc without uniform or have two rc behavior (but I don't think there is a use case yet with true rc).

Reply:
Ok, makes sense. We should probably rework commit overlay, so that it also includes RC changes. In `get` If for some key the RC change delta in he overlay is negative, we query the backend for the actual counter value.